docs/pypi-replication.md

```diff
@@ -24,81 +25,126 @@ P_30                  	all	0.3102
 
 ## MS MARCO Passage Ranking
 
+Before we can evaluate the output, we need to first fetch the eval script and relevance judgments:
+
+```bash
+$ wget https://raw.githubusercontent.com/castorini/anserini-tools/master/scripts/msmarco/msmarco_eval.py
```

Comment:
This url gives `404 Not Found`

Reply:
do'h, I just changed this: https://github.com/castorini/anserini-tools/pull/23

Will fix.